integration/aggregate_compat_test.go

```diff
@@ -712,6 +716,202 @@ func TestAggregateCompatGroupCount(t *testing.T) {
 	testAggregateStagesCompat(t, testCases)
 }
 
+func TestAggregateCompatGroupSum(t *testing.T) {
+	providers := shareddata.AllProviders().
+		// skipped due to https://github.com/FerretDB/FerretDB/issues/2185.
+		Remove("Composites").
+		Remove("ArrayStrings").
+		Remove("ArrayInt32s").
+		Remove("Mixed").
+		Remove("ArrayAndDocuments").
+		// TODO: handle $sum of doubles near max precision.
+		// https://github.com/FerretDB/FerretDB/issues/2300
+		Remove("Doubles")
```

Comment:
So we prefer using Remove(). Because newly added providers will be tested unless specifically removed.

Reply:
I wonder if we could modify `Remove` to use object identities (pointers / memory addresses in that case) instead of names (that we could change and break this code):
```go
Remove(shareddata.Doubles)
```